executor/inspection_result.go

```diff
@@ -82,9 +82,9 @@ type (
 	// version in the cluster
 	versionInspection struct{ inspectionName }
 
-	// currentLoadInspection is used to check the current load of memory/disk/cpu
+	// nodeLoadInspection is used to check the node load of memory/disk/cpu
```

Comment:
nodeLoadInspection is used to check the load of memory/disk/cpu of a node.

Reply:
great, done.